### ISSUE TITLE
Returning empty lyrics.

### ISSUE THREAD

**Issue by mokoshalb** [NTag/lyrics.ovh#10]
Tried to figure out what the cause it but couldn't detect it.

**Comment by antgobar** [comment]
Same here, this seems to happen when I request the same artist/song in relatively quick succession.
e.g. I call for Coldplay/Yellow it returns the lyrics as expected.
If I do this a few seconds later, lyrics return empty. 
I've not timed this accurately but I found that about a minute or two, I can call the same artist/song and again get lyrics.
Maybe it is some in built throttling to prevent many requests?
Certainly a bit frustrating when wanting to test my code..!

**Comment by ggasmithh** [comment]
This is also happening to me, and at an interval much longer than a few seconds.

**Comment by antgobar** [comment]
Yeah I have to wait about 1-2 minutes. I've added a 'time check' function to my code which prevents fetching lyrics if less than 2 minutes have passed between the same searches, though there's still some issues..

**Comment by simplenotezy** [comment]
Happens to me too!

**Comment by simplenotezy** [comment]
Even happens on their main website:
https://user-images.githubusercontent.com/3786627/103087882-43ac6480-45e9-11eb-97ac-7f6e9942c171.mov

**Comment by realmilandev** [comment]
Same here, and sometimes it loads, and then goes away.

**Comment by AkashRajpurohit** [comment]
@NTag Any update on this?

**Comment by gabrielgrenier** [comment]
Same, I have to wait 2min +

**Comment by shanjng** [comment]
Same here this is frustrating because this is a straightforward lyrics API (which is a valuable thing)

**Comment by christinna9031** [comment]
Some lyrics are completely missing. I can select them from the list on lyrics.ovh, but I get back empty lyrics each time. Blinding Lights - The Weeknd for example never returns any lyrics.

**Comment by NTag** [comment]
I mostly rewrote the website and changed the sources, it should work better now.
